src/test/java/com/exasol/cloudwatch/TestConstants.java

```diff
@@ -1,6 +1,6 @@
 package com.exasol.cloudwatch;
 
 public class TestConstants {
-    public static final String EXASOL_DOCKER_DB_VERSION = "7.0.5";
-    public static final String LOCAL_STACK_IMAGE = "localstack/localstack:0.12.6";
+    public static final String EXASOL_DOCKER_DB_VERSION = "7.1.3";
+    public static final String LOCAL_STACK_IMAGE = "localstack/localstack:0.12.9";
```

Comment:
Upgrading to latest version is not possible due to issue https://github.com/localstack/localstack/issues/5115